cpp/src/io/json/json_gpu.cu

```diff
@@ -16,23 +16,23 @@
 
 #include "json_common.h"
 #include "json_gpu.h"
+#include "thrust/detail/copy.h"
```

Comment:
What is the rule on using quotes instead of `< >` for include? I think this line should be using `< >`
In other projects we have used quotes only for including header files within the _component_ which was a bit subjective.

Reply:
The rule is "use them whenever you realize vscode automatically added headers for you." :)

Anecdotally, `"..."` is used for same-directory imports, or relative-path imports. I'm not an expert, and would rather use `<...>` when available. :shrug: 

Thanks for pointing these out!